Cargo.toml

```diff
@@ -37,6 +37,10 @@ std = ["serde", "bitvec/std", "byte-slice-cast/std"]
 bit-vec = ["bitvec"]
 fuzz = ["std", "arbitrary"]
 
+# Make error fully descriptive with chaining error message.
+# Should not be used in constrained environment.
+chain-error = []
```

Comment:
Yeah it should be enabled by the default feature.

Reply:
but should "std" also enable it ?

Reply:
Sorry, brain malfunction. I had forgotten that we have a `std` feature :P Yeah, please add it to the `std` feature.

Reply:
yes done